.hgsigs

```diff
@@ -0,0 +1 @@
+4902098137874f15c8e538b73e87446c6742bad9 0 iQEcBAABAgAGBQJYv0qvAAoJEKMv031G/7TTMZYIAJX+thhjZQ5hfz54kPWYg5kjtR9yjccRQwHnAFHFuPFcHZNlM71RXfdMxkNfnq8TCX6WG/0dqwkARnSMgWqVuA7qMBp8WTYjvjHLMNxMPS7c2eIlTNK00ReGfg+Pr0jD6dUa/yr3PYxlyWwvPzbwmzIyFUs7GfhDmmzJwAzLUjtv/qAw9EwXOgz1u8VGmflx5jAn8dMz8HvWci1qG8otmx9XoMGlKouB1GZ0cIFVeEZ8NS7fRvEW0dhP0qhLVGApvbZF6ewAw+seqHeaj/ibVVEVTXTCR5IvpnqXwU7yUsrVqyr8HQZdV+SAl7hL0aWQ9S/n4gn2Zhe1fIQWx0MQhLE=
```

Comment:
I don't think this Mercurial stuff belongs into the Git repository :)
Perhaps add it to Mercurial's equivalent of the ".gitignore" file to prevent accidentally committing it again.